omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint_required_recipe.lua.erb

```diff
@@ -0,0 +1,28 @@
+<% unless node['private_chef']['required_recipe']['enable'] -%>
```

Comment:
Since the route won't be rendered if `node['private_chef']['required_recipe']['enable']` is false, is this block necessary?